go.mod

```diff
@@ -9,14 +9,4 @@ require (
 	github.com/stretchr/testify v1.7.0
 )
 
-require (
-	github.com/Azure/azure-sdk-for-go/sdk/azcore v0.20.0 // indirect
-	github.com/Azure/azure-sdk-for-go/sdk/internal v0.8.2 // indirect
-	github.com/davecgh/go-spew v1.1.1 // indirect
-	github.com/microsoft/kiota/authentication/go/azure v0.0.0-20211201125630-3501743a5dc5 // indirect
-	github.com/pmezard/go-difflib v1.0.0 // indirect
-	github.com/yosida95/uritemplate/v3 v3.0.1 // indirect
-	golang.org/x/net v0.0.0-20211123203042-d83791d6bcd9 // indirect
-	golang.org/x/text v0.3.7 // indirect
-	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
-)
+require github.com/microsoft/kiota/serialization/go/json v0.0.0-20220118084409-ae624d0bd384
```

Comment:
it seems your IDE trimmed kiota auh azure. I've noticed that happening sometimes when using some commands. I think it's because we don't have a direct dependency on it. I'd like to keep it there so people don't have to pull it independently, what do you think?
Otherwise we need to update this https://github.com/microsoftgraph/msgraph-sdk-go/#1-installation and that https://docs.microsoft.com/en-us/graph/sdks/sdk-installation#install-the-microsoft-graph-go-sdk-preview

Reply:
Yeah, it was trimmed after I ran `go mod tidy` which will remove any installed package that is not referenced in the source code. Let's not fight the tooling. What do I need to update in the references you shared?

Reply:
So if I understand your comment correctly you'd remove the reference and update the documentation?

Reply:
Yes.

Reply:
alright, please go ahead with the PRs then :)

Reply:
https://github.com/microsoftgraph/msgraph-sdk-go/pull/69